src/components/TextField/SbTextField.vue

```diff
@@ -76,7 +76,7 @@
         />
       </SbTooltip>
       <SbIcon
-        v-if="(iconRight || error) && type !== 'password'"
+        v-if="(iconRight || (error && iconRight)) && type !== 'password'"
```

Comment:
I'd love to see this as a computed property 😬

Reply:
Done! ❤️